### DISCUSSION TITLE
Notification on mention

### DISCUSSION THREAD

**Discussion by nick-traeger** [YetiForceCompany/YetiForceCRM#15502] [Feature improvement]
Hello,
you can mention an employee in the comments of tickets using @Name, for example, and YetiForce then links the mentioned employee. Is it possible that this employee then receives a notification by email so that he can react if necessary?

**Comment by vovpff** [comment]
@nick-traeger I think this functionality unavailable in system. You can implement it by yourself

**Reply by rskrzypczak** [reply]
Yes, there is no such functionality at the moment
